### PR TITLE
Product foundry update with latest dev

### PR DESCRIPTION
**Pull request by cbordier**
Thank you for your contribution to the Amplify-Central repo.

## Describe the changes

Update product foundry documentation based on latest implementation

## Checklist for contributors

Before submitting this PR, please make sure:

* [X] You have read the [contribution guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/)
* [X] You have signed the [Axway CLA](https://cla.axway.com/)
* [X] You have verified the technical accuracy of your change
* [X] You have verified that your change does not expose any sensitive information (passwords, keys, etc.)
* [X] You have followed the [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)  (unless this is is a Netlify CMS contribution)

_Put an x in the boxes that apply. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your change._

### CODE REVIEW

**Comment by cbordier** [comment]
@lbadenhop, I think I caught up the documentation for product foundry.
Can you review.

**Comment by lbadenhop** [comment]
@cbordier Please review my changes. 
In "Archive a product," if I select the product from the list view, then it takes me to the Product Information page where I do not have access to the more options menu (ellipsis). I had to change the wording. Is "Delete a product version" correct? I could not test the instructions.    
Also, to get to the product details from Central, the instructions state: 
1. Navigate to *Product Foundry*.
2. Select **Products > All Products** and open the product details. 
However, the actual steps are: 
1. Navigate to *Product Foundry*.
2. Select **All Products** and open the product details.

File: content/en/docs/manage_product_foundry/foundry_product_management.md [hunk 1]
```diff
@@ -52,7 +53,7 @@ View the following detailed information for a specific product in any state (Dra
 
 * Product Name - The display name for the product in the WebUI
 * Product state of the selected version
-* Product version - The most recent version available. Could be empty if product has not been released
+* Product version - The most recent version available. Coan be empty if product is not released
```
Comment (robertengaxway) [inline content/en/docs/manage_product_foundry/foundry_product_management.md:56]: typo 'Coan'

**Review by robertengaxway — APPROVED** [review]
Found one typo but rest looks good

**Comment by cbordier** [comment]
@lbadenhop final review done. You can merge.

**Pull request merged**